packages/web/app/providers/wallet/wallet-wrapper-provider.tsx

```diff
@@ -56,33 +59,45 @@ const WalletWrapperImplementationProvider: React.FC<Props> = ({ children }) => {
   const [{ chains: onboardChains, connectedChain }, setOnboardChain] =
     useSetChainOptions;
 
-  const wallet = React.useMemo(() => W3O.getWallet(onboardWallet), [onboardWallet]);
+  const wallet = React.useMemo(
+    () => W3O.getWallet(onboardWallet),
+    [onboardWallet]
+  );
 
   const isWalletConnected = !!wallet;
 
-  const status = React.useMemo(() => W3O.getStatus(isWalletConnected, connecting), [isWalletConnected, connecting]);
+  const status = React.useMemo(
+    () => W3O.getStatus(isWalletConnected, connecting),
+    [isWalletConnected, connecting]
+  );
 
-  const chains = React.useMemo(() => W3O.getAvailableChains(
-      onboardChains.length === 0 ? [defaultChain as W3OChain] : onboardChains
-    ), [onboardChains]);
+  const chains = React.useMemo(
+    () =>
+      W3O.getAvailableChains(
+        onboardChains.length === 0 ? [defaultChain as W3OChain] : onboardChains
+      ),
+    [onboardChains]
+  );
 
-  const chain = React.useMemo(() => W3O.getCurrentChain(chains, connectedChain?.id), [connectedChain?.id, chains]);
+  const chain = React.useMemo(
+    () => W3O.getCurrentChain(chains, connectedChain?.id),
+    [connectedChain?.id, chains]
+  );
 
-  const provider = React.useMemo(() => onboardWallet?.provider
-      ? W3O.getProvider(onboardWallet?.provider)
-      : null, [onboardWallet?.provider]);
+  const provider = React.useMemo(
+    () =>
+      onboardWallet?.provider ? W3O.getProvider(onboardWallet?.provider) : null,
+    [onboardWallet?.provider]
+  );
 
   const connect = React.useCallback(async () => {
     const success = await W3O.connect(onboardConnect);
-    if (success) 
+    if (success)
       await W3O.autoSelectProperChain(chain, chains, setOnboardChain);
-    
   }, [chain, chains, onboardConnect, setOnboardChain]);
 
   const disconnect = React.useCallback(async () => {
-    if (wallet) 
-      await W3O.disconnect(wallet.label, onboardDisconnect);
-    
+    if (wallet) await W3O.disconnect(wallet.label, onboardDisconnect);
```

Comment:
Eh, the linter must not allow such code.

Reply:
yeah, it should wrap it by curly braces :))